internal/operator/nodeagent/dep/health/dep.go

```diff
@@ -104,9 +104,10 @@ Type=simple
 User=root
 ExecStart=/usr/local/bin/health --http %s %s
 Restart=always
-MemoryMax=20M
 MemoryLimit=20M
 RestartSec=10
+CPUAccounting=yes
+MemoryAccounting=yes
```

Comment:
you sure?

> MemoryLimit=bytes
Specify the limit on maximum memory usage of the executed processes. The limit specifies how much process and kernel memory can be used by tasks in this unit. Takes a memory size in bytes. If the value is suffixed with K, M, G or T, the specified memory size is parsed as Kilobytes, Megabytes, Gigabytes, or Terabytes (with the base 1024), respectively. Alternatively, a percentage value may be specified, which is taken relative to the installed physical memory on the system. If assigned the special value "infinity", no memory limit is applied. This controls the "memory.limit_in_bytes" control group attribute. For details about this control group attribute, see Memory Resource Controller.

> Implies "MemoryAccounting=yes".

> This setting is deprecated. Use MemoryMax= instead.

https://www.freedesktop.org/software/systemd/man/systemd.resource-control.html

Reply:
we should just keep an eye on this, but i think with centos7 it is not yet a issue